pavelib/assets.py

```diff
@@ -760,8 +760,9 @@ def webpack(options):
     static_root_lms = Env.get_django_setting("STATIC_ROOT", "lms", settings=settings)
     static_root_cms = Env.get_django_setting("STATIC_ROOT", "cms", settings=settings)
     config_path = Env.get_django_setting("WEBPACK_CONFIG_PATH", "lms", settings=settings)
+    debug = Env.get_django_setting("DEBUG", "lms", settings=settings)
     environment = 'NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms}'.format(
-        node_env="production" if settings != Env.DEVSTACK_SETTINGS else "development",
+        node_env="development" if debug is True else "production",
```

Comment:
I think checking the DEBUG setting does the job but any option that is not hardcoding a check on the name of the settings file would do the job:

- "development" if config_path == "webpack.dev.config.js" else "production"
- adding another setting "WEBPACK_CONFIG_TYPE"
- ...

WDYT?